docs/yul_language/statements.rst

```diff
@@ -0,0 +1,118 @@
+Statements
+==========
+
+For-loop
+________
+
+For-loops are completely supported in Solang. ``continue`` and ``break`` statements are also supported.
```

Comment:
some words about how for loops are written would be helpful (the init, cond, next block/expression)